packages/reactotron-server/src/server/schema/connection.ts

```diff
@@ -0,0 +1,88 @@
+import { ObjectType, Field, Int } from "type-graphql"
+
+@ObjectType()
+export class Connection {
```

Comment:
Making this a more complex object is on my hit list (In this case a more complex object means a simpler object for the consumer. I plan to combine some things into sub types so this isn't so flat)

Reply:
Ya, this is my fault.  I threw them all in there without any thought of organizing.

Can we leave them as is so we can try to ride out the current protocol for now?  One thing I love doing is using the existing reactotron in real projects, but swapping out the "app/backend" for the new version.

I understand we'll have to change the protocol, but I'd like to hold off as long as possible.

Reply:
My plan was to transform this object to a more complex one in the server here so `core-server` and `core-client` stay the same (for now). My hope is to not touch anything other then the new package so we are technically backward compatible for now.

Reply:
great plan!  always 1 step ahead of me.